samples/nrf9160/serial_lte_modem/src/slm_util.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef SLM_UTIL_
+#define SLM_UTIL_
+
+/**@file slm_util.h
+ *
+ * @brief Utility functions for serial LTE modem
+ * @{
+ */
+
+#include <zephyr/types.h>
+#include <ctype.h>
+#include <stdbool.h>
+
+/**
+ * @brief Compare string ignoring case
+ *
+ * @param str1 First string
+ * @param str2 Second string
+ * @param str2_len Length of second string
+ *
+ * @retval true If two commands match, false if not.
```

Comment:
s/commands/strings

Reply:
And you should use `@return` in all functions in this file, `@retval` is for listing return error codes.

Reply:
Updated